content/en/docs/ops/best-practices/security/index.md

```diff
@@ -42,6 +42,80 @@ For example, in the [authorization for HTTP traffic task](/docs/tasks/security/a
+
+The enforcement point of authorization policy is the Envoy proxy which is different from the actual resource access point that is
+usually in the backend application. A policy mismatch happens when the Envoy proxy and the backend application interprets the request
+differently.
+
+The mismatch leads to either unexpected rejection or policy bypass. The latter is usually a security incident that needs to be
+fixed immediately, and it's also why we need path normalization in the authorization policy.
+
+For example, you have an authorization policy to reject requests with path `/data/secret`. A request with path `/data//secret` will
+not be rejected because it does not match with the path defined in the authorization policy due to the extra forward slash `/` in the path.
+
+The request goes through and later the backend application returns the same response that it returns for the path `/data/secret`
+because the backend application normalizes the path `/data//secret` to `/data/secret` as it considers the double forward slashes
+`//` is equivalent to a single forward slash `/`.
+
+In this example, the policy enforcement point (Envoy proxy) had a different understanding of the path than to the resource access
+point (backend application). The different understanding caused the mismatch and subsequently the bypass of the authorization policy.
+
+This becomes a complicated problem because of the following factors:
+
+* Lacking of a clear standard of the normalization.
+
+* Backends and frameworks in different layers have their own special normalization.
+
+* Applications can even have arbitrary normalizations for its own use cases.
+
+Istio authorization policy implements built-in support of various basic normalization options to help you to better address
+the problem:
+
+* Read the [Guideline on configuring the path normalization option](/docs/ops/best-practices/security/#guideline-on-configuring-the-path-normalization-option)
+  to understand which normalization option you should use.
+
+* Read the [Customize your system on path normalization](/docs/ops/best-practices/security/#customize-your-system-on-path-normalization) to
+  understand the detail of each normalization option.
+
+* Read the [Less common normalization configurations](/docs/ops/best-practices/security/#less-common-normalization-configurations) for
+  alternative solutions in case you need any unsupported normalization options.
+
+### Guideline on configuring the path normalization option
+
+#### Case 1: You do not need normalization at all
+
+Before diving into the details of configuring normalization, you should first make sure if you actually need any of the normalizations.
+
+You do not need the normalization if you do not use authorization policy in the first place or if your authorization policy does not
+use any `path` fields.
+
+You may not need the normalization if all your authorization policies follow the safer authorization pattern which the worst
```

Comment:
```suggestion
You may not need the normalization if all your authorization policies follow the safer authorization pattern which, in the worst
```
If you didn't add the "Safer Authorization Policy Pattern" bullet point earlier, I would add a link here to that section.